src/components/NumericalInput/index.tsx

```diff
@@ -65,8 +62,10 @@ export const Input = React.memo(function InnerInput({
       {...rest}
       value={value}
       onChange={event => {
-        enforcer(event.target.value)
+        // replace commas with periods, because uniswap exclusively uses period as the decimal separator
+        enforcer(event.target.value.replace(/,/g, '.'))
```

Comment:
it could also help to have a pattern attribute on the input
https://www.w3schools.com/tags/att_input_pattern.asp